pkg/controllers/management/k3sbasedupgrade/k3sbased_upgrade_handler.go

```diff
@@ -17,8 +16,8 @@ import (
 )
 
 const (
-	K3sAppNme  = "rancher-k3s-upgrader"
-	Rke2AppNme = "rancher-rke2-upgrader"
+	K3sAppName  = "rancher-k3s-upgrader"
+	Rke2AppName = "rancher-rke2-upgrader"
```

Comment:
Oh, this is causing problems in the CI. All the references to them need to be corrected.

Reply:
Updated